daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md

```diff
@@ -94,6 +94,73 @@ In this example, at trigger time, which is `@every 1s` according to the `Schedul
 
 At the trigger time, the `prodDBBackupHandler` function is called, executing the desired business logic for this job at trigger time. For example:
 
+### Triggered Job Handling Assumptions
+
+#### HTTP
+
+When you create a job using Dapr's Jobs API, Dapr will automatically assume there is an endpoint available at 
+`/job/<job-name>`. For instance, if you schedule a job named `test`, Dapr expects your application to listen for job 
+events at `/job/test`. Ensure your application has a handler set up for this endpoint to process the job when it is 
+triggered. For example:
+
+```go
```

Comment:
Can you add a note that his example uses Go but is applicable to any language?

Reply:
sure thing

Reply:
done. lmk if you want me to tweak it

Reply:
Thanks, I ended up changing it directly. We'll soon need to edit this again anyway as Java/Dotnet support is coming :)

Reply:
Added .NET bits with #4385 

I don't know if the very bottom section is needed calling out the various SDK approaches given that we'll have just listed them in the above codetabs.